Sources/BraintreeShopperInsights/BTButtonOrder.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+/// The order or ranking in which payment buttons appear.
```

Comment:
We can likely add the warnings to all of these new enums as well, just to safely mark them as beta/that they may change